astar.py

```diff
@@ -34,16 +37,18 @@ def weight(node1, node2):
   c2 = node2.getCoords()
   t1 = getThickness(c1)
   t2 = getThickness(c2)
-  return (t1+t2)/2
+  return 2**(t1+t2)

```

Comment:
Whitespace is typically preferred between operators for readability. Write this as `2 ** (t1 + t2)`